packages/twenty-front/src/modules/object-record/field/meta-types/display/components/PipelineStepsDisplay.tsx

```diff
@@ -0,0 +1,9 @@
+import { Status } from '@/ui/display/status/components/Status';
+
+import { usePipelineStepsField } from '../../hooks/usePipelineStepsField';
+
+export const PipelineStepsDisplay = () => {
+  const { fieldValue } = usePipelineStepsField();
+
+  return <Status color={fieldValue.color} text={fieldValue.label} />;
```

Comment:
the fieldValue should be a string (we only want to store the value). The color and the label should be retrieved through metadata (see how Select is implemented)

Reply:
> the fieldValue should be a string (we only want to store the value). The color and the label should be retrieved through metadata (see how Select is implemented)

Sorry for that as I haven't checked the latest update.